scripts/globals/conquest.lua

```diff
@@ -994,6 +995,11 @@ end
 -----------------------------------
 
 xi.conquest.overseerOnTrade = function(player, npc, trade, guardNation, guardType)
+
+    if xi.garrison.onTrade(player, npc, trade, guardNation) then
```

Comment:
Since garrison and conquest are so tightly coupled, I've opted to put the calls to the garrison stuff in these global helpers for the overseers

Happy to hear other solutions

Reply:
Reasonable, and I wouldn't even complain if they were just baked into conquest.

Reply:
wait.. wasn't this exactly what you argued against when you spoke with Runaway about it 2 months ago? 
![image](https://user-images.githubusercontent.com/40971475/189016330-3633049f-0b00-4cd4-8634-78e1d6895822.png)

Reply:
Ah, I wasn't aware I wasn't allowed to change my mind in the face of a massively out-of-date and difficult to rebase piece of work, which I'm going to have to go through by hand to get over the finish line. I'll revert these commits, sorry!